### ISSUE TITLE
Wrong export of Construction subclasses

### ISSUE THREAD

**Issue by gioagu** [3dcitydb/energy-ade-citydb#5]
This might be a direct consequence of the bug issues before with the LayerComponent.

For testing purposes, I add here for your convenience a sample dataset containing 1 building with nearly everything you may add with the KIT Energy profile :-)

Constructions are referenced via Xlinks by the ThermalBoundaries.

[OneBuilding_KIT.zip](https://github.com/3dcitydb/energy-ade-citydb/files/6385057/OneBuilding_KIT.zip)

**Comment by clausnagel** [comment]
Wow, this is indeed helpful. Snoke's palace on Alderaan? ;-)

I imported and exported your dataset after fixing #4. The constructions are referenced from the thermal boundaries both in your test dataset and in the exported dataset. So, looks fine for me. Or am I missing something?

I found a possible issue with `gml:TimePeriod`. Here is an example from your dataset:

```xml
<gml:TimePeriod>
  <gml:beginPosition>00:00:00</gml:beginPosition>
  <gml:endPosition>23:59:00</gml:endPosition>
  <gml:duration>P0Y0M1D</gml:duration>
</gml:TimePeriod>
```

The current database schema does not support storing the `gml:duration` value. We skipped this in agreement with KIT because it was neither used nor required for the KIT datasets. Thus, this value gets lost when importing. If you think it's necessary, we have to adapt the database schema.

**Comment by gioagu** [comment]
Thanks again for the quick reply. And congrats for the 4.3 release!

> Wow, this is indeed helpful. Snoke's palace on Alderaan? ;-)
1) Well, Alderaan was blown up in Ep IV: no one will ever be able to really check if I made it up...

> <gml:TimePeriod>
>   <gml:beginPosition>00:00:00</gml:beginPosition>
>   <gml:endPosition>23:59:00</gml:endPosition>
>   <gml:duration>P0Y0M1D</gml:duration>
> </gml:TimePeriod>

> The current database schema does not support storing the `gml:duration` value. 
2) I realised that it should actually be 23:59:*59*
3) Good point. As a matter of fact, gml:duration is not required and can/could be derived from the begin and end position. I'd say, for now keep the db as it is.

**Comment by clausnagel** [comment]
Ok, great. I plan to release a new package containing the bugfixes as soon as possible. Let me know if you stumble across other things.

**Comment by clausnagel** [comment]
just released version [1.2.1](https://github.com/3dcitydb/energy-ade-citydb/releases/tag/v1.2.1)
